baremetal/metal3machine_manager.go

```diff
@@ -1836,7 +1837,8 @@ func (m *MachineManager) getMatchingNodesWithoutLabelCount(ctx context.Context,
 	err = m.duplicateProviderIDsExist(validNodes, providerIDLegacy, providerIDNew)
 	if err != nil {
 		// There are, at least, two nodes. Details are in err
-		return 2, err
+		nodes := 2
```

Comment:
This feels dumb IMO, can we just nolint this one?

Reply:
I agree